src/series.rs

```diff
@@ -54,7 +54,7 @@ pub enum Msg {
 
 /// Describes how to process each item of series data
 #[derive(Clone, PartialEq)]
-pub enum SeriesType {
+pub enum Type {
```

Comment:
This might be more intuitive as `PlotType` rather than just `Type`.

Reply:
It can be qualified using `series::Type` so I think we're good. I've minimised a number of names now given module namespacing.